guava/src/com/google/common/base/Preconditions.java

```diff
@@ -1330,8 +1332,8 @@ public static void checkState(
    * @throws IllegalArgumentException if {@code size} is negative
    */
   @CanIgnoreReturnValue
-  public static int checkElementIndex(int index, int size) {
-    return checkElementIndex(index, size, "index");
+  public static @NonNegative int checkElementIndex(@NonNegative int index, @NonNegative int size) {
+    return checkElementIndex(index, size, "index"); // This comment contains @AnnotatedFor so that this line is ignored by count-suppressions
```

Comment:
Please remove all comments like this, to keep the diffs small.

We use the `count-suppressions` script only to estimate which bugs/features to add to the Index Checker, but not for any statistics in a paper.  (Or, that should be true if it is not currently!)
I could try to improve the regex in `count-suppressions` if this is too big an annoyance during development.

Reply:
It is used by the script print-fps.sh in the reproduction package.

Reply:
Although the to comment improves the numbers slightly, I don't think this should be merged into master because it would pollute the source code.

How about changing `count-suppressions.sh` to filter out lines that end with `;`?  Can you try that and see if it changes the numbers?  If not, we could remove these comments and make that change to `count-suppressions.sh` instead.  (If necessary, leave this pull request open for the ISSTA reviewers, and make a new one that eliminates the extraneous comments.)